src/main/java/aquality/appium/mobile/configuration/SwipeConfiguration.java

```diff
@@ -0,0 +1,91 @@
+package aquality.appium.mobile.configuration;
+
+import aquality.selenium.core.utilities.ISettingsFile;
+import com.google.inject.Inject;
+
+import java.time.Duration;
+
+public class SwipeConfiguration implements ISwipeConfiguration {
+    private final int retries;
+    private final Duration timeout;
+    private final double horizontalSwipeTopPointXCoefficient;
+    private final double horizontalSwipeBottomPointXCoefficient;
+    private final double horizontalSwipeTopPointYCoefficient;
+    private final double horizontalSwipeBottomPointYCoefficient;
+    private final double verticalSwipeLeftPointXCoefficient;
+    private final double verticalSwipeRightPointXCoefficient;
+    private final double verticalSwipeLeftPointYCoefficient;
+    private final double verticalSwipeRightPointYCoefficient;
+
+    @Inject
+    public SwipeConfiguration(ISettingsFile settingsFile) {
+        this.retries = Integer.parseInt(settingsFile.getValue("/swipe/retries").toString());
+        this.timeout = Duration.ofSeconds(Long.parseLong(settingsFile.getValue("/swipe/timeout").toString()));
+        this.horizontalSwipeTopPointXCoefficient = Double.parseDouble(
+                settingsFile.getValue("/swipe/horizontalSwipeTopPointXCoefficient").toString());
+        this.horizontalSwipeBottomPointXCoefficient = Double.parseDouble(
+                settingsFile.getValue("/swipe/horizontalSwipeBottomPointXCoefficient").toString());
```

Comment:
why do we casting toString and then parsing as double? If the value is double in settings file, we should be able to cast object to double directly

Reply:
As far as I know, in JSON there is no `Double` primitive: only `Number` and `String`. 
Probably that causes the situation when `settingsFile.getValue("/swipe/horizontalSwipeTopPointXCoefficient")` returns `String`. 
That means that if we will cast the value directly to double, we'll receive: `Error injecting constructor, java.lang.ClassCastException: java.lang.String cannot be cast to java.lang.Double`

Reply:
probably it's an issue witho our JsonSettingsFile implementation:
https://github.com/aquality-automation/aquality-selenium-core-java/blob/master/src/main/java/aquality/selenium/core/utilities/JsonSettingsFile.java#L49
Okay, we can leave it as is for now

Reply:
can we make up with some private method like 'private double getSwipeCoefficient(String name)` ?

Reply:
parts `"settingsFile.getValue("/swipe/"` and `).toString());` are still in common